persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/StructuralVariantMapper.xml

```diff
@@ -0,0 +1,90 @@
+        structural_variant.SITE2_ENTREZ_GENE_ID AS "${prefix}site2EntrezGeneId",
+        gene2.HUGO_GENE_SYMBOL AS "${prefix}site2HugoSymbol",
+        structural_variant.SITE2_ENSEMBL_TRANSCRIPT_ID AS "${prefix}site2EnsemblTranscriptId",
+        structural_variant.SITE2_EXON AS "${prefix}site2Exon",
+        structural_variant.SITE2_CHROMOSOME AS "${prefix}site2Chromosome",
+        structural_variant.SITE2_POSITION AS "${prefix}site2Position",
+        structural_variant.SITE2_DESCRIPTION AS "${prefix}site2Description",
+        structural_variant.SITE2_EFFECT_ON_FRAME AS "${prefix}site2EffectOnFrame",
+        structural_variant.DNA_SUPPORT AS "${prefix}dnaSupport",
+        structural_variant.RNA_SUPPORT AS "${prefix}rnaSupport",
+        structural_variant.NORMAL_READ_COUNT AS "${prefix}normalReadCount",
+        structural_variant.TUMOR_READ_COUNT AS "${prefix}tumorReadCount",
+        structural_variant.NORMAL_VARIANT_COUNT AS "${prefix}normalVariantCount",
+        structural_variant.TUMOR_VARIANT_COUNT AS "${prefix}tumorVariantCount",
+        structural_variant.NORMAL_PAIRED_END_READ_COUNT AS "${prefix}normalPairedEndReadCount",
+        structural_variant.TUMOR_PAIRED_END_READ_COUNT AS "${prefix}tumorPairedEndReadCount",
+        structural_variant.NORMAL_SPLIT_READ_COUNT AS "${prefix}normalSplitReadCount",
+        structural_variant.TUMOR_SPLIT_READ_COUNT AS "${prefix}tumorSplitReadCount",
+        structural_variant.ANNOTATION AS "${prefix}annotation",
+        structural_variant.BREAKPOINT_TYPE AS "${prefix}breakpointType",
+        structural_variant.CENTER AS "${prefix}center",
+        structural_variant.CONNECTION_TYPE AS "${prefix}connectionType",
+        structural_variant.EVENT_INFO AS "${prefix}eventInfo",
+        structural_variant.CLASS AS "${prefix}variantClass",
+        structural_variant.LENGTH AS "${prefix}length",
+        structural_variant.COMMENTS AS "${prefix}comments",
+        structural_variant.EXTERNAL_ANNOTATION AS "${prefix}externalAnnotation",
+        structural_variant.DRIVER_FILTER AS "${prefix}driverFilter",
+        structural_variant.DRIVER_FILTER_ANNOTATION AS "${prefix}driverFilterAnn",
+        structural_variant.DRIVER_TIERS_FILTER AS "${prefix}driverTiersFilter",
+        structural_variant.DRIVER_TIERS_FILTER_ANNOTATION AS "${prefix}driverTiersFilterAnn"
+    </sql>
+
+    <select id="fetchStructuralVariants" resultType="org.cbioportal.model.StructuralVariant">
+        SELECT 
+        <include refid="select">
+            <property name="prefix" value=""/>
+        </include> 
+        FROM structural_variant
+        JOIN genetic_profile ON structural_variant.GENETIC_PROFILE_ID = genetic_profile.GENETIC_PROFILE_ID
+        JOIN gene gene1 ON structural_variant.SITE1_ENTREZ_GENE_ID = gene1.ENTREZ_GENE_ID
+        JOIN gene gene2 ON structural_variant.SITE2_ENTREZ_GENE_ID = gene2.ENTREZ_GENE_ID
+        <if test="studyIds.size() > 0 and sampleIds.size() > 0">
```

Comment:
please remove this `if`. The join should always be made since sample id and cancer study ids should be in the response (e.g. in the scenario where one queries without specifying sample ids or cancer study, we return for all samples and studies right? )